src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -79,6 +99,22 @@ public CaseSearchResult searchCases(
         return result;
     }
 
+    private List<String> getCaseTypeIds(List<String> caseTypeIds) {
+        if (isAConsolidationQuery(caseTypeIds)) {
+            return getCaseTypes();
+        }
+        return caseTypeIds;
```

Comment:
what about if the client has IDAM role `caseworker-caa` and request is `ctid=*`? 
In that case is the search happening on all the existing case types?

Reply:
This feature has been implemented now.